### PR TITLE
Inv-280 Create OpenMRS 2-Compatible Department Page

### PR DESCRIPTION
**Pull request by k-joseph**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/openhmis/inventory/api/IDepartmentDataService.java [hunk 1]
```diff
@@ -13,6 +13,8 @@
  */
 package org.openmrs.module.openhmis.inventory.api;
```
Comment (k-joseph) [inline api/src/main/java/org/openmrs/module/openhmis/inventory/api/IDepartmentDataService.java]: Changes to the service layer are not used and i can therefore revert them

File: api/src/main/java/org/openmrs/module/openhmis/inventory/api/impl/DepartmentDataServiceImpl.java [hunk 2]
```diff
@@ -13,6 +13,10 @@
  */
 package org.openmrs.module.openhmis.inventory.api.impl;
```
Comment (k-joseph) [inline api/src/main/java/org/openmrs/module/openhmis/inventory/api/impl/DepartmentDataServiceImpl.java]: Changes to the service layer are not used and i can therefore revert them

**Comment by akmad** [comment]
This doesn't appear to build on our build server while the Institution change does. Are there still some changes needed to this PR?

**Comment by akmad** [comment]
This code was moved into our angular framework and has now been superseded. Thanks for this @k-joseph!

**Pull request closed without merging**